### PR TITLE
Provide a fallback for null display names.

### PR DESCRIPTION
**Pull request by ben-eb**
I think it would be nice to provide a better output for when a display name hasn't been filled in. Currently, we get this output which is not ideal;

https://github.com/ben-eb/postcss-discard-empty/blob/0abd5bd271056243a72d72399c2885b149b36f31/README.md#contributors

What do you think? 😄

### CODE REVIEW

File: lib/contributors/github.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = function getUserInfo(username, cb) {
     var body = JSON.parse(res.body);
     var user = {
       login: body.login,
-      name: body.name,
+      name: body.name === null ? username : body.name,
```
Comment (jfmengels) [inline lib/contributors/github.js]: Do you know if users can make their name be an empty string? If so, maybe `bode.name || username` would be a better fit?

**Comment by jfmengels** [comment]
Huh, didn't think of that case. Sounds good to me, thanks!

I'll let you respond and/or makes changes linked to my comment, and I'll merge after that.

**Comment by ben-eb** [comment]
Done. https://github.com/jfmengels/all-contributors-cli/pull/21/commits/abb0b087b1e8765909585e4cdf274609d02a21e3

**Comment by jfmengels** [comment]
:+1: Thanks a lot!

**Comment by jfmengels** [comment]
Released in v3.0.6

**Pull request merged**